wrappers/kn/README.md

```diff
@@ -40,6 +42,48 @@ barcodeReader.read(image).joinToString("\n") { barcode: Barcode ->
 
 Here you have to load your image into memory by yourself and pass the decoded data to the constructor of `ImageView`.
 
+### Writing
+
+A trivial use case looks like this:
+
+```kotlin
+import zxingcpp.BarcodeFormat
+import zxingcpp.BarcodeWriter
+import zxingcpp.CreatorOptions
+import zxingcpp.ImageFormat
+import zxingcpp.ImageView
+import zxingcpp.toSVG
+import zxingcpp.toImage
+
+val text: String = "Hello, World!"
+val format = BarcodeFormat.QRCode
+val width: Int = 256
+val height: Int = 256
+
+val opts = CreatorOptions().apply {
+    format = BarcodeFormat.QRCode
+   // more options, see documentation
+}
+
+val barcode = Barcode(text, opts)
+// or
+val barcode2 = Barcode(text, format)
+
+val barcodeWriter = BarcodeWriter().apply {
```

Comment:
I'm still not convinced that `BarcodeWriter` brings any value here. `WriterOptions` that can optionally be passed to `Barcode.toImage()` seems pretty adequate to me.

Reply:
done